org.sugarlabs.MusicBlocks.json

```diff
@@ -0,0 +1,125 @@
+{
+    "app-id": "org.sugarlabs.MusicBlocks",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "branch": "stable",
+    "sdk": "org.freedesktop.Sdk",
+    "base": "io.atom.electron.BaseApp",
+    "base-version": "stable",
+    "command": "run.sh",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=network",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--device=dri",
+        "--filesystem=home",
+        "--env=MUSICBLOCKS_PATH=/app/share/musicblocks"
+    ],
+    "build-options" : {
+        "env": {
+            "NPM_CONFIG_LOGLEVEL": "info"
+        }
+    },
+    "modules": [
+        {
+            "name": "nodejs",
+            "cleanup": [
+                "/include",
+                "/share",
+                "/app/lib/node_modules/npm/changelogs",
+                "/app/lib/node_modules/npm/doc",
+                "/app/lib/node_modules/npm/html",
+                "/app/lib/node_modules/npm/man",
+                "/app/lib/node_modules/npm/scripts"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
```

Comment:
(Also for how to use zypak.)

Reply:
Thanks! As a point of clarification, Music Blocks is really only a static HTML page rather than a full-fledged electron app – so it should hopefully look more like the simple example here: https://github.com/flatpak/flatpak-builder-tools/tree/master/node/vanilla-quick-start. I'm still having problems with something not being stored that is needed in electron postinstall as well as some SUID sandbox helper binary misconfiguration, but I'll try using node12 to see if that fixes anything.